### PR TITLE
(PA-6886) Add DigiCertGlobalRootG2 to Solaris image

### PR DESCRIPTION
**Pull request by AriaXLi**

### CODE REVIEW

File: setup/common/003_solaris_cert_fix.rb [hunk 1]
```diff
@@ -79,6 +104,10 @@
   on(host, 'chmod a+r /root/DigiCert_Trusted_Root_G4.pem')
   on(host, 'cp -p /root/DigiCert_Trusted_Root_G4.pem /etc/certs/CA/')
   on(host, 'rm /root/DigiCert_Trusted_Root_G4.pem')
+  on(host, "echo '#{DigiCertGlobalRootG2}'> /etc/openssl/certs/DigiCertGlobalRootG2.pem")
```
Comment (joshcooper) [inline setup/common/003_solaris_cert_fix.rb:107]:
```suggestion
create_remote_file(host, 'DigiCert_Trusted_Root_G2.pem', DigiCertG2)
```

File: setup/common/003_solaris_cert_fix.rb [hunk 2]
```diff
@@ -79,6 +104,10 @@
...
+  on(host, 'chmod a+r /etc/openssl/certs/DigiCertGlobalRootG2.pem')
```
Comment (joshcooper) [inline setup/common/003_solaris_cert_fix.rb:108]:
```suggestion
  on(host, 'chmod a+r /root/DigiCertGlobalRootG2.pem')
```

File: setup/common/003_solaris_cert_fix.rb [hunk 3]
```diff
@@ -79,6 +104,10 @@
...
+  on(host, 'ln -s /etc/openssl/certs/DigiCertGlobalRootG2.pem /etc/openssl/certs/607986c7.0')
```
Comment (joshcooper) [inline setup/common/003_solaris_cert_fix.rb:109]:
```suggestion
  on(host, 'cp -p /root/DigiCert_Trusted_Root_G4.pem /etc/certs/CA/')
  on(host, 'rm /root/DigiCert_Trusted_Root_G4.pem')
```

File: setup/common/003_solaris_cert_fix.rb [hunk 4]
```diff
@@ -79,6 +104,10 @@
...
+  on(host, '/usr/sbin/svcadm restart /system/ca-certificates')
```
Comment (joshcooper) [inline setup/common/003_solaris_cert_fix.rb:110]: Since the service is restarted below, we don't need to do that here:

```suggestion
```

**Comment by AriaXLi** [comment]
Verified this change worked by running it on puppet-agent#main ad-hoc:
https://jenkins-platform.delivery.puppetlabs.net/view/puppet-agent/view/ad-hoc/job/platform_puppet-agent-extra_puppet-agent-modules-and-pez_adhoc-ad_hoc/309/

**Review by cthorn42 — APPROVED** [review]

**Pull request merged**